README.md

```diff
@@ -144,6 +146,7 @@ Apache License, v 2.0 except as noted otherwise in the [LICENSE](./LICENSE) file
 [python.org downloads]: https://www.python.org/downloads/
 [installing virtualenv]: https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#installing-virtualenv
 [installing pip]: https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#installing-pip
-[SDK documentation]: TODO
+[SDK documentation]: https://data-attribute-recommendation-python-sdk.readthedocs.io/en/latest/
+[SDK security guide]: ./docs/source/security.rst
 [github issues]: TODO
```

Comment:
Can you also add:

```suggestion
[github issues]: https://github.com/SAP/data-attribute-recommendation-python-sdk/issues
```

Reply:
Yes, completely missed this. I've added this now. Thanks!